torchvision/datasets/kinetics.py

```diff
@@ -23,4 +24,7 @@ def __getitem__(self, idx):
         video, audio, info, video_idx = self.video_clips.get_clip(idx)
         label = self.samples[video_idx][1]
 
+        if self.transform is not None:
```

Comment:
@bjuncek @soumith Looking for your opinions on how we should call the video transforms.

Indeed, we might have both `video_transform` and `audio_transform`, given that the dataset returns both data.

Would you prefer to stick with `transform` for meaning `video_transform`, or choose a different name?

Reply:
For this PR, I think we should keep this as transform, and maybe have a wrapper for audio transforms, or wait until batched transforms for both audio and video are ready